### DISCUSSION TITLE
New functionalitys

### DISCUSSION THREAD

**Discussion by osevan** [roddhjav/apparmor.d#248] [General]
https://www.phoronix.com/news/Linux-6.7-AppArmor

Lib io_uring mediation

**Comment by roddhjav** [comment]
This will be part of a new version of apparmor (4.x). It will include other nice feature such as user namespace, mqueue, and extended network filter. It it currently in alpha, and you need a special kernel to have all feature.

All of these new feature, will obviously be used into this project.

I you look deeper into this project, you can see that here is [already initial support](https://github.com/roddhjav/apparmor.d/blob/4df3f2e52f846d66dd9bf0e45dce4063e315005d/pkg/prebuild/build.go#L90) in the build system for apparmor 4. The idea is to be able to convert all `abi/4.0` to the (future) old `abi/3.0` for distribution that won't support apparmor 4 yet.
